packages/synapse-react-client/src/mocks/MockSynapseContext.tsx

```diff
@@ -25,12 +25,10 @@ export const MOCK_CONTEXT = createContext(MOCK_CONTEXT_VALUE)
  *
  * If using @testing-library/react, see {@link TestingLibraryUtils#createWrapper}
  */
-export const SynapseTestContext = jest
```

Comment:
This may be used in Jest or Vitest, so we cannot make it a Jest Mocked Function. That's ok, it doesn't need to be a mock at all.